README.md

```diff
@@ -53,8 +68,7 @@ In one sentence, explain why.
 
 ## A1.2
 
-«replace this with your answer»
-
+O(n), since esentially it is determining a "palindrome" persay, so the maximum is n/2 because it may be false before then, and since n/2 is still greater than a linear search, it is still O(n). 
```

Comment:
# -1

The O(n) is right, but remember we're talking about memory here, and any memory used to parse this is in a stack, so that should be the focus of the answer.